mne/viz/tests/test_ica.py

```diff
@@ -44,8 +44,9 @@ def _get_epochs():
     raw = _get_raw()
     events = _get_events()
     picks = _get_picks(raw)
-    epochs = Epochs(raw, events[:10], event_id, tmin, tmax, picks=picks,
-                    baseline=(None, 0))
+    with warnings.catch_warnings(record=True):  # bad proj
+        epochs = Epochs(raw, events[:10], event_id, tmin, tmax, picks=picks,
+                        baseline=(None, 0))
```

Comment:
IIRC no not in this situation, Epochs does picking internally and then
constructs proj so it can't be prevented. Maybe by picking from Raw first
but I think I tried it at one point